src/Microsoft.Framework.PackageManager/Utils/LockFileUtils.cs

```diff
@@ -37,11 +40,11 @@ internal static class LockFileUtils
 
             foreach (var framework in frameworks)
             {
-                var group = new LockFileFrameworkGroup();
+                var group = new Microsoft.Framework.Runtime.DependencyManagement.LockFileFrameworkGroup();
                 group.TargetFramework = framework;
 
-                IEnumerable<PackageDependencySet> dependencySet;
-                if (VersionUtility.TryGetCompatibleItems(framework, package.DependencySets, out dependencySet))
+                IEnumerable<NuGet.PackageDependencySet> dependencySet;
+                if (NuGet.VersionUtility.TryGetCompatibleItems(framework, package.DependencySets, out dependencySet))
```

Comment:
See DNU REFACTORING TODO above. This is the old implementation temporarily used by `dnu publish` for incremental refactoring. We will remove this after we refactor `dnu publish` later. The new implementation of this method can be found below and `dnu restore` is already using it. Later we'll make `dnu publish` use the new implementation, too.